### DISCUSSION TITLE
TypeScript 관련 질문

### DISCUSSION THREAD

**Discussion by nuyeel00** [naverz/zepeto-studio-kor#603] [5. Q&A]
오늘 처음 제페토 템플릿을 설치하고 가이드를 따라가보고 있는데 TypeScript에서 바로 막혔습니다. 새로 test 라는 이름으로 파일을 생성하면 이렇게 원래 알던 스크립트 아이콘이 아닌 비주얼 스튜디오 아이콘으로 생성되지만 누르면 수정은 정상적으로 되긴 합니다.
![image](https://user-images.githubusercontent.com/103350993/162604502-08bbd267-45f9-4b77-b11f-9c41f9af541b.png)

그런데 인스펙터에서 이렇게 뜨고...
![image](https://user-images.githubusercontent.com/103350993/162604535-629168b7-2ebc-4e7d-9b8e-9f5ed3b7d700.png)

빈 오브젝트에 스크립트를 넣어보려고 해도 추가가 되지 않아요ㅠㅠ 제가 뭔가 놓친 부분이 있는 걸까요? 아무리 다시 생성해봐도 똑같아서... 샘플 스크립트랑 똑같은 .ts 파일이고 딱히 뭔가 놓쳤다기엔 이제 막 켜서 저거 하나 생성해본 게 다라서요... 아주 기본적인 문제 같은데 혹시 해결 방법이 있을까요? 감사합니다ㅜㅜ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 
유니티 환경 설정창 Preferences > External Tools에서 에디터가 어떤 것으로 설정되어 있는지 확인해 보세요.
![스크린샷 2022-04-11 오전 11 07 38](https://user-images.githubusercontent.com/100118650/162653437-1033cda0-f8dd-434d-9bb4-a2683475f841.png)

Typescript 에디터로는 VSCode (무료) 또는 Rider(유료)를 추천 합니다.
VSCode를 사용하실 경우 아래 가이드에 맞춰서 버전 선택도 해주시면 됩니다. 
https://github.com/naverz/zepeto-studio-kor/discussions/481

**Reply by nuyeel00** [reply]
이거 외에도 주르륵 문제가 있었는데 다 해결했습니다 감사합니다...ㅠㅠ!!
